TcUnit-Runner/VisualStudioInstance.cs

```diff
@@ -107,29 +107,56 @@ public void Close()
             loaded = false;
         }
 
-        private void LoadDevelopmentToolsEnvironment(string visualStudioVersion)
+        private void LoadDevelopmentToolsEnvironment(string visualStudioVersion, bool pinnedTCVersion)
         {
             /* Make sure the DTE loads with the same version of Visual Studio as the
              * TwinCAT project was created in
              */
             
             // Load the DTE
             string VisualStudioProgId = VisualStudioDteAvailable(visualStudioVersion);
-
+            bool versionAvailable = false;
             dte.UserControl = false; // have devenv.exe automatically close when launched using automation
             dte.SuppressUI = true;
             // Make sure all types of errors in the error list are collected
             dte.ToolWindows.ErrorList.ShowErrors = true;
             dte.ToolWindows.ErrorList.ShowMessages = true;
             dte.ToolWindows.ErrorList.ShowWarnings = true;
+            // First set the SilentMode and then try to open the Remote Manager
+            var tcAutomationSettings = dte.GetObject("TcAutomationSettings");
+            tcAutomationSettings.SilentMode = false; // Only available from TC3.1.4020.0 and above
```

Comment:
Why silentMode = false?

Reply:
It must be true, my mistake. 
It is important that Silent Mode is set to true before you try to start the Remote Manager
Otherwise the programme will stop because a message box appears.